lib/classes/Variables.js

```diff
@@ -822,6 +827,17 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    const groups = variableString.match(this.strToBoolRefSyntax);
+    const param = groups[1];
+    if (param === 'true') {
+      return BbPromise.resolve(true);
+    } else if (param === 'false') {
+      return BbPromise.resolve(false);
+    }
+    return BbPromise.resolve(true);
```

Comment:
Here, I think we should follow with one of below options:
- Throw exception (as provided string didn't reflect boolean string) - _that might to restrictive_
- Return `null` (as no matching boolean counterpart was found)
- Apply language boolean coercion (so e.g. empty strings, or empty non string values (e.g. `null`) are resolved as `false`)

Reply:
I made updates. Let me know what you think.